src/keycloak.ts

```diff
@@ -293,6 +333,7 @@ export class KeyCloak extends Construct {
 
     const region = cdk.Stack.of(this).region;
     const regionIsResolved = !cdk.Token.isUnresolved(region);
+    const { cpu = 2048, memoryLimitMiB =4096, dbClusterInstances = 2 } = props;
```

Comment:
Isn't that overkill for the dev deployment?

Reply:
these are just the default values, currently present in the production, and these can be overridden in the actual stack definition :slightly_smiling_face: 
But sure, the parameters are way too exaggerated for the dev environment!